cmake.deps/CMakeLists.txt

```diff
@@ -12,12 +12,15 @@ list(APPEND CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/cmake" "${PROJECT_SOURCE_DI
 include(CheckCCompilerFlag)
 include(Util)
 
+set(CMAKE_FIND_FRAMEWORK "LAST" CACHE STRING "How find_* commands choose between macOS Frameworks and unix-style package components.")
+
 set(DEPS_CMAKE_ARGS
   -D CMAKE_C_COMPILER=${CMAKE_C_COMPILER}
   -D CMAKE_C_STANDARD=99
   -D CMAKE_GENERATOR=${CMAKE_GENERATOR}
   -D CMAKE_GENERATOR_PLATFORM=${CMAKE_GENERATOR_PLATFORM}
-  -D CMAKE_POSITION_INDEPENDENT_CODE=ON)
+  -D CMAKE_POSITION_INDEPENDENT_CODE=ON
+  -D CMAKE_FIND_FRAMEWORK=${CMAKE_FIND_FRAMEWORK})
```

Comment:
I'm wondering if there's a problem with always setting this to `LAST` aka skipping the cache variable? I don't have mac experience so please correct me.

Reply:
You could try, but that has the potential to surprise some users. I guess we could set it and see if someone complains.

Reply:
Hmm, I understand. Should we revert this part of your changes and only set this in the CI? Sorry for the back and forth, trying to come up with a solution that is both maintainable and useful.

Reply:
Sure, no problem. It's all small changes anyway.

Reply:
Just realised we need to keep this to make sure our setting of `CMAKE_FIND_FRAMEWORK` propagates to any dependent builds.